third_party/gems/BUILD

```diff
@@ -0,0 +1 @@
+licenses(["notice"])
```

Comment:
What does this have to do with the rest of the PR?

Reply:
Absolutely nothing. Bazel requires files in third_party to have a BUILD file with a `licenses` declaration.